index.js

```diff
@@ -10,11 +10,9 @@ const obj = () => Object.create(null);
 
 // Prevent caching of this module so module.parent is always accurate
 delete require.cache[__filename];
-const parentDir = path.dirname(module.parent.filename);
```

Comment:
I would prefer to still keep this here and add the `.` here instead.

You can use https://github.com/sindresorhus/require-uncached to be able to test it.